docs/changelog.rst

```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+* :release:`2.1.0 <2021-08-25>`
+* :feature:`7163` Add a Matrix federation checker tool.
+* :feature:`7162` Don't prune routes anymore.
+* :feature:`7196` Send encrypted secret with the LockedTransfer message, obviating the need for SecretRequest and SecretReveal messages. This reduces the time needed to complete a payment.
+* :feature:`7195`, `7247` Enable WebRTC support. We now keep exactly one open WebRTC channel per peer, and regularly check for channel availability.
+* :feature:`7233`, `7264` Add support for cooperative settlement. If both parties are online and able to settle cooperatively, settlement is performed in a much quicker fashion. Closing a channel will now automatically try doing cooperative settlement first.
+* :feature:`7276` We don't pin Synapse to 1.33.0 anymore.
```

Comment:
`Remove pin Synapse to 1.33.0.` or similar

Reply:
How about "Synapse is not pinned to 1.33.0 anymore"?